### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by Vinay-Khanagavi** [Vinay-Khanagavi/Cyberbear.github.io#16] [Ideas]
### Share your ideas and design

**Comment by olegcherev** [comment]
Hi,

What do you think of gamification of the contribution process? Commits/PRs/Issues rewarded with XP/Levels/Badges, contributors featured on Leaderboard, maintainers create challenges etc.

I made a GitHub app to implement those elements, and it looks complimentary to the idea of the gamification of the cyber awareness.

I'd appreciate your feedback for [devActivity](https://github.com/apps/devactivity-app)
There's a [Live Demo](https://demo.devactivity.com/achievements/?activeTab=achievements&organizationId=1) (no auth required) and [Public Dashboard Demo](https://demo.devactivity.com/public/?organizationLogin=demo-organization&organizationId=1)

**Reply by Vinay-Khanagavi** [reply]
That's cool @olegcherev , That might help a lot of organisation which have like too many contributors

**Comment by Vinay-Khanagavi** [comment]
https://discord.gg/mdRn98xw

**Comment by Dipeshdahiya** [comment]
![image](https://github.com/user-attachments/assets/22cad8f6-1d86-41d9-a953-ea72f3afcb00)
Here the spacing between the rows is very less , let me adjust the spacing

**Reply by Vinay-Khanagavi** [reply]
@Dipeshdahiya  sorry currently landing page is not for open source you can work the main project. thank you
